### PR TITLE
Details added to existing operations

### PR DESCRIPTION
**Pull request by mshkolnik22**
## Description

User is able to see details of an existing operation

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

Self check in a browser.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by argaudreau — CHANGES_REQUESTED** [review]
I'd like to see more details about the operation, specifically all the settings used in the Create Operation window, like:
* Jitter
* Fact source
* Group
* Parser
* Auto close
* Visibility
* etc...
This kind of info would be best placed in a modal or popup, which would be opened via a button since space is limited on the line with the operation name. I'd recommend taking a look at what is returned from the GET request to get the current operation to see what kind of metadata it returns so you can get an idea of what to display.

**Review by argaudreau — CHANGES_REQUESTED** [review]
Just one small change, but otherwise looks good to me!

File: templates/operations.html [hunk 1]
```diff
@@ -36,9 +36,14 @@ <h2>Operations</h2>
         </form>
 
         <template x-if="selectedOperation && selectedOperationID">
-            <div class="section p-0 pb-2">
-                <div>
-                    <h3 x-text="selectedOperation.name"></h3>
+            <div class="section p-2 pb-2">
+                <div class="is-flex-direction-row p-2 mb-2">
+                    <span x-text="'Current Operation:  ' + selectedOperation.name + '  |  ' + selectedOperation.adversary.name + '  |  ' + selectedOperation.source.name" ></span>
```
[the remark this answers is not in the document]
Comment (mshkolnik22) [reply templates/operations.html]: implemented.

**Comment by mshkolnik22** [comment]
I made all the requested changes. Can you please take another look?

**Review by argaudreau — CHANGES_REQUESTED** [review]
The modal appears off in Firefox:
Screen Shot 2022-06-27 at 2 31 48 PM

The More Details button might look better to the left of the download button, and if we move it there renaming it to "Operation Details" would be more explicit. The button as-is seems kind of jarring next to the text, so moving it to the button row below that would clean it up.

**Review by ZacharyLPalmer — COMMENTED** [review]
Concur with adam on the button moving inline with download/delete. I looked at the modal in firefox and it didn't look off, so not sure why it's showing up like that for Adam. I would also say that having the operation name in line with the other information might make more sense, but I think it works either way.

**Review by argaudreau — APPROVED** [review]

**Review by ZacharyLPalmer — APPROVED** [review]
Looks good!

**Pull request merged**